README.rst

```diff
@@ -53,11 +53,17 @@ Installation
     These can also be specificied per field by passing the ``public_key`` or
     ``private_key`` parameters to ``ReCaptchaField`` - see field usage below.
 
-#. If you require a proxy, add a ``RECAPTCHA_PROXY`` setting, for example:
+#. If you require a proxy, add a ``RECAPTCHA_PROXY`` setting dictionary:
```

Comment:
setting (dictionary of proxies), for example

Reply:
@cblignaut minor correction: that is a set, not a dictionary.

Reply:
@AltusBarry how does the code know which proxy to use? (I'll read a bit further. MAybe I get the answer myself.)

Reply:
@cblignaut Apologies. I see it _should_ be a dictionary, but is incorrectly defined as a set below, hence my comment.